docs/sources/configuration/parquet.md

```diff
@@ -1,76 +1,67 @@
 ---
 title: Parquet
+menuTitle: Apache Parquet
 weight: 75
 ---
 
-# Apache Parquet backend
+# Apache Parquet block format
 
-<span style="background-color:#f3f973;">This is an experimental feature released with [Tempo 1.5]({{< relref "../release-notes/v1-5/" >}}). For more information about how to enable it, continue reading.</span>
 
-Tempo now has a columnar block format based on Apache Parquet.
+Tempo has a default columnar block format based on Apache Parquet. [TraceQL]({{< relref "../traceql" >}}), the query language for traces, requires Parquet. In a future release, Parquet will be required to use traces search.
+
 A columnar block format may result in improved search performance and also enables a large ecosystem of tools access to the underlying trace data.
 
 For more information, refer to the [Parquet design document](https://github.com/mdisibio/tempo/blob/design-proposal-parquet/docs/design-proposals/2022-04%20Parquet.md) and [Issue 1480](https://github.com/grafana/tempo/issues/1480).
 
+If you install using the new Helm charts, then Parquet is enabled by default.
+
 ## Considerations
 
-The new Parquet block format can be used as a drop-in replacement for Tempo's existing block format.
-No data conversion or upgrade process is necessary.
-As soon as the Parquet format is enabled, Tempo starts writing data in that format, leaving existing data as-is.
+The new Parquet block is enabled by default in Tempo 2.0. No data conversion or upgrade process is necessary. As soon as the Parquet format is enabled, Tempo starts writing data in that format, leaving existing data as-is.
+
+The new Parquet block format requires more CPU and memory resources than the previous v2 format but provides faster search and the new TraceQL.
 
-Please note, however, that enabling the Parquet block format means Tempo will require more CPU and memory resources than it previously did. 
+## Convert to Parquet
```

Comment:
This command is only applicable to advanced users running an earlier parquet schema and requiring certain performance or troubleshooting.  Let's move it below the other content, or even remove it altogether?   For those users schema conversion happens automatically, and there is no need for this command or conversion at all for users running 2.0/Parquet for the first time.

Reply:
I moved this section to the bottom of the page with a note that it might be useful for people who used Parquet prior to Tempo 2.0